conf/application.conf

```diff
@@ -190,6 +190,7 @@ mongodbURI = "mongodb://127.0.0.1:27017/clowder"
 # All requests to extractors and any other external process is send
 # using rabbitmq, this will setup the connection information.
 clowder.rabbitmq.uri="amqp://guest:guest@localhost:5672/%2f"
+clowder.primary=true
```

Comment:
this is in the middle of the rabbitmq section, probably better to put it in its own area and add some text what this does (mention that it is only needed in case you use multiple instances of clowder).

Reply:
Took care of this.